definitions/EiffelConfidenceLevelModifiedEvent/3.4.0.yml

```diff
@@ -0,0 +1,184 @@
+        type: array
+        items:
+          $ref: ../EiffelCustomDataProperty/1.0.0.yml
+    required:
+      - name
+      - value
+    additionalProperties: false
+  links:
+    type: array
+    items:
+      $ref: ../EiffelEventLink/1.1.1.yml
+required:
+  - meta
+  - data
+  - links
+additionalProperties: false
+_links:
+  CAUSE:
+    description: 'Identifies a cause of the event occurring. SHOULD
+      not be used in conjunction with __CONTEXT__: individual events
+      providing __CAUSE__ within a larger context gives rise to ambiguity.
+      It is instead recommended to let the root event of the context
+      declare __CAUSE__.'
+    required: false
+    multiple: true
+    targets:
+      any_type: true
+      types: []
+  CONTEXT:
+    description: Identifies the activity or test suite of which this
+      event constitutes a part.
+    required: false
+    multiple: false
+    targets:
+      any_type: false
+      types:
+        - EiffelActivityTriggeredEvent
+        - EiffelTestSuiteStartedEvent
+  FLOW_CONTEXT:
+    description: 'Identifies the flow context of the event: which is
+      the continuous integration and delivery flow in which this occurred
+      – e.g. which product, project, track or version this is applicable
+      to.'
+    required: false
+    multiple: true
+    targets:
+      any_type: false
+      types:
+        - EiffelFlowContextDefinedEvent
+  SUBJECT:
+    description: Identifies a subject of the confidence level; in other
+      words, what the confidence level applies to.
+    required: true
+    multiple: true
+    targets:
+      any_type: false
+      types:
+        - EiffelArtifactCreatedEvent
+        - EiffelArtifactDeployedEvent
+        - EiffelCompositionDefinedEvent
+        - EiffelSourceChangeCreatedEvent
+        - EiffelSourceChangeSubmittedEvent
+  SUB_CONFIDENCE_LEVEL:
+    description: 'Used in events summarizing multiple confidence levels.
+      Example use case: the confidence level "allTestsOk" summarizes
+      the confidence levels "unitTestsOk, "scenarioTestsOk" and "deploymentTestsOk",
+      and consequently links to them via __SUB_CONFIDENCE_LEVEL__.
+      This is intended for purely descriptive, rather than prescriptive,
+      use. Future editions might deprecate this link in favour of CONFIDENCE_BASIS.'
+    required: false
+    multiple: true
+    targets:
+      any_type: false
+      types:
+        - EiffelConfidenceLevelModifiedEvent
+  CONFIDENCE_BASIS:
+    description: 'Used to declare the basis for which confidence statement(s) this event have used. 
+      The __CAUSE__ link tells what caused the event sending whereas __CONFIDENCE_BASIS__ declares the reason for
+      selecting the provided `data.name` and/or `data.value`.'
+    required: false
+    multiple: true
+    targets:
+      any_type: false
+      types:
+        - EiffelConfidenceLevelModifiedEvent
+        - EiffelTestCaseFinishedEvent
+        - EiffelTestSuiteFinishedEvent
```

Comment:
Sorry for noticing this only now, but would you mind moving up this link type so that they're in alphabetical order? I suspect we're sorting them when the Markdown is rendered but having them alphabetical in the source file would make things more navigable.

Also, we generally use `__fieldname__` to reference other field names, not backticks. Compare output of `git grep '<backtick>data\.' -- definitions` with `git grep '__data\.' -- definitions` (replace <backtick> with an actual backtick obviously; I couldn't get the formatting right here).

Reply:
No we don't sort it as far as I could see but both issues are fixed in [ebf4416](https://github.com/eiffel-community/eiffel/pull/368/commits/ebf4416bbd3caa91e0d935ec12da2210730a092f)